### PR TITLE
Step8

### PR DESCRIPTION
**Pull request by andriimikhaylovskiy**

### CODE REVIEW

File: src/main/java/mate/academy/spring/boot/controller/BookController.java [hunk 1]
```diff
@@ -32,6 +33,7 @@
 public class BookController {
     private final BookService bookService;
 
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
```
Comment (Vasul-Pankiv) [inline src/main/java/mate/academy/spring/boot/controller/BookController.java]:
```suggestion
    @PreAuthorize("hasRole('ADMIN')")
```

Comment (Vasul-Pankiv) [reply src/main/java/mate/academy/spring/boot/controller/BookController.java]: fix it everywhere

**Review by Vasul-Pankiv — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/spring/boot/service/BookServiceImpl.java [hunk 2]
```diff
@@ -63,10 +63,17 @@ public void deleteById(Long id) {
...
     @Transactional
     public BookDto updateBook(Long id, CreateBookRequestDto requestDto) {
-        Book updateBook = bookMapper.updateBookFromDto(
-                bookRepository.findById(id).orElseThrow(() ->
-                        new EntityNotFoundException("Can't get book by id = " + id)), requestDto);
-        return bookMapper.toDto(updateBook);
+        String isbn = null;
+        Book oldBook = bookRepository.findById(id).orElseThrow(() ->
+                new EntityNotFoundException("Can't get book by id = " + id));
+        if (requestDto.isbn() == null || requestDto.isbn().isBlank()) {
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/spring/boot/service/BookServiceImpl.java]: Does CreateBookRequestDto have some validations for isbn field like NotBlank? 
Also, your Book entity should has constraint for isbn like nullable = false and unique = true
Check these please, and remove redundant logic if needed

File: src/main/resources/db/changelog/changes/01_1-insert-default-book.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+                  value: 'Default book'
+              - column:
+                  name: author
+                  value: 'Default author'
+              - column:
+                  name: isbn
+                  value: '000-00-00000000'
+              - column:
+                  name: price
+                  value: '1.99'
```
Comment (Elena-Bruyako) [inline src/main/resources/db/changelog/changes/01_1-insert-default-book.yaml:20]:
```suggestion
                  value: '1.99'
                  
```

Comment (Elena-Bruyako) [reply src/main/resources/db/changelog/changes/01_1-insert-default-book.yaml:20]: Add empty line in the end for all changelogs

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Looks good, let's fix some comments

**Review by Vasul-Pankiv — APPROVED** [review]

**Pull request merged**